src/recipes/messaging/config.js

```diff
@@ -208,7 +217,7 @@ module.exports = {
     },
     "release": {
       restdays: 30,
-      sample: 10/million,  // 1 in 100000
+      sample: 100/million,  // 1 in 10000
```

Comment:
I know we're probably decommissioning Monday, but minor comment:
If we're going with a higher sample I would say 10 * 10/million and add a comment.  Feel free to ignore.